src/io/flutter/vmService/ServiceExtensions.java

```diff
@@ -132,6 +164,8 @@
     slowAnimations,
     togglePlatformMode,
     toggleSelectWidgetMode,
+    toggleOnDeviceWidgetInspector,
+    enableOnDeviceInspector,
```

Comment:
nit: looks like this list is alphabetized. Put enableOnDeviceInspector in correct position

Reply:
done